model/src/main/java/com/basistech/rosette/dm/Mention.java

```diff
@@ -0,0 +1,229 @@
+/*
+* Copyright 2014 Basis Technology Corp.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+package com.basistech.rosette.dm;
+
+import com.google.common.base.Objects;
+
+import java.util.Map;
+
+/**
+ * A mention of a entity in the text.  For example, "George" and
+ * "George Washington" are mentions of type "PERSON".
+ */
```

Comment:
Do we want to add the `@since` javadoc tag for new classes (and methods)?
(Fine to say "no".)